docs/configuration/radio/device.mdx

```diff
@@ -11,24 +11,34 @@ The device config options are: Role, Serial Output, and Debug Log. Device config
-| `LOST_AND_FOUND` |                                                                                                                                                                                                  Lost and Found - Used to automatically send a text message with current position at frequent intervals to the primary channel for the device: "I'm lost! Position: lat / long"                                                                                                                                                                                                   |
+### Roles
+
+| Device Role    | Description                                                                                                              | Best Uses                                                                                                                              |
+| -------------- | ------------------------------------------------------------------------------------------------------------------------ | -------------------------------------------------------------------------------------------------------------------------------------- |
+| CLIENT         | Standard device for communication over the network, supports client apps.                                                | General use for individuals needing to communicate over the Meshtastic network with support for client applications.                   |
+| CLIENT_MUTE    | Same as CLIENT but does not forward packets from other devices.                                                          | Situations where a device needs to participate in the network without assisting in packet routing, reducing network load.              |
+| CLIENT_HIDDEN  | Only broadcasts as needed for stealth or power savings, still rebroadcasts known meshes.                                 | Use in clandestine operations or to reduce airtime/power consumption while still participating in the network.                         |
+| TRACKER        | Broadcasts GPS position packets as priority.                                                                             | Tracking the location of individuals or assets, especially in scenarios where timely and efficient location updates are critical.      |
+| LOST_AND_FOUND | Broadcasts location to default channel regularly for recovery efforts.                                                   | Used for recovery efforts of a lost device.                                                                                            |
```

Comment:
I'd suggest something like "broadcasts a location message" so we know it sends a text, not just a location packet

Reply:
"Broadcasts location as message to default channel regularly for recovery efforts."